packages/lexicon/src/User/index.ts

```diff
@@ -0,0 +1,36 @@
+import { Brand } from 'src/Tools';
+import { GUID } from 'src/GUID';
+
+export type User = {
+  applicationId: ApplicationId;
+  birthdate: Date;
+  email: Email;
+  email_verified: boolean;
+  family_name: string;
+  given_name: string;
+  roles: string[];
+  scope: string;
+  sid: SId;
+  sub: Sub;
+  tid: TId;
+};
+
+export type ApplicationId = Brand<GUID, 'ApplicationId'>;
+export type SId = Brand<GUID, 'SId'>;
+export type Sub = Brand<GUID, 'Sub'>;
+export type TId = Brand<GUID, 'TId'>;
+export type Email = Brand<string, 'Email'>;
+
+// const exampleUser: User = {
```

Comment:
Is this comment meant to be here?

Reply:
Yeah, I thought it might be useful to show how a example. 👍